libtelemetry/src/full/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -476,4 +477,16 @@ public Thread newThread(Runnable runnable) {
       };
     }
   }
+
+  @SuppressWarnings("WeakerAccess")
+  public void setBaseUrl(String eventsHost) {
+    if (isValidUrl(eventsHost)) {
+      telemetryClient.setBaseUrl(eventsHost);
+    }
+  }
+
+  private boolean isValidUrl(String eventsHost) {
```

Comment:
should this method be `static`?

Reply:
it is used in a single non-static method, so no need to make it static.